yarn.lock

```diff
@@ -1574,7 +1574,7 @@
     "@wdio/types" "7.13.2"
     p-iteration "^1.1.8"
 
-"@webcomponents/webcomponentsjs@^2.5.0":
+"@webcomponents/webcomponentsjs@^2.5.0", "@webcomponents/webcomponentsjs@^2.6.0":
```

Comment:
this should also disappear, not suppose to update the yarn.lock file

Reply:
Done